omero/sysadmins/version-requirements.txt

```diff
@@ -160,7 +160,7 @@ Components
 
 * Ice
 
-  * [5.1] 3.4 deprecated, 3.5 recommended, 3.6 not supported
+  * [5.1] 3.4 deprecated, 3.5 recommended, 3.6 supported
```

Comment:
This "supported" may end up needing to be "experimental" depending on how far we get with the rest of https://trello.com/c/OHLViGl8/47-ice-3-6